backend/mock/service/awsmock/awsmock.go

```diff
@@ -92,6 +93,10 @@ func (s *svc) TerminateInstances(ctx context.Context, region string, ids []strin
 	return nil
 }
 
+func (s *svc) S3StreamingGet(ctx context.Context, region string, bucket string, key string) (io.ReadCloser, error) {
+	return nil, nil
```

Comment:
either panic, return nil, err or return a fake file. nil nil is not a valid value could cause mock server to behave funny or crash unexpectedly.